packages/drivers/src/driver/pgsql/index.ts

```diff
@@ -94,7 +95,9 @@ export default class PostgreSQL extends AbstractDriver<Pool, PoolConfig> impleme
           };
         });
       })
-      .catch(err => ([{
+      .catch(err => {
+        cli.release();
```

Comment:
```suggestion
        cli && cli.release();
```
Check if cli is set before calling release